spack/packages/dla-future/package.py

```diff
@@ -25,11 +28,19 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
 
     variant("miniapps", default=False, description="Build miniapps.")
 
+    variant("c", default=False, description="Build C API")
```

Comment:
Always compile the standard C API and make the ScaLAPACK API optional. Call this variant `+scalapack`.